### PR TITLE
Add a `gramex check` command

### PR DESCRIPTION
**Pull request by jaidevd**

### CODE REVIEW

File: gramex/apps/ui/__init__.py [hunk 1]
```diff
@@ -133,7 +133,8 @@ def sass2(handler, path: str = join(ui_dir, 'gramexui.scss')):
         with open(scss_path, 'w', encoding='utf-8') as handle:
             handle.write('\n'.join(content))
         # Compile SASS file. Allow @import from template dir, UI dir, Bootstrap dir
-        proc = yield gramex.service.threadpool.submit(subprocess.run, [
+        # proc = yield gramex.service.threadpool.submit(subprocess.run, [
+        proc = subprocess.run([
```
Comment (sanand0) [inline gramex/apps/ui/__init__.py:137]: - Rename sass2 to _sass without the coroutine
- Make sass2 yield `gramex.service.threadpool.submit(_sass, ...)`

File: gramex/gramex.yaml [hunk 2]
```diff
@@ -109,7 +109,7 @@ log:
             - logfile
     loggers:
         gramex:
-            level: DEBUG
+            level: INFO
```
Comment (sanand0) [inline gramex/gramex.yaml:112]: I added this. Please remove.

File: gramex/install.py [hunk 3]
```diff
@@ -773,3 +780,69 @@ def license(args, kwargs):
...
+        capture = Capture(engine='chrome')
+        image = io.BytesIO()
+        image.write(capture.jpg('https://python.org'))
+        image.seek(0)
+        assert Image.open(image).format == 'JPEG'
+
+    def _check_r():
+        try:
+            from gramex.ml import r
+            assert r('sum(c(1, 2, 3))')[0] == 6, 'rpy2 fail.'
```
Comment (sanand0) [inline gramex/install.py:822]: Remove assert message

File: gramex/apps/ui/__init__.py (hunk 1 above)
Comment (jaidevd) [reply gramex/apps/ui/__init__.py:136]: Done, @sanand0 . Please take a look.

**Pull request closed without merging**